core/state_processor.go

```diff
@@ -302,19 +304,19 @@ func (p *StateProcessor) Process(block *types.Block, etxSet types.EtxSet) (types
 				if err := gp.SubGas(params.CallValueTransferGas); err != nil {
 					return nil, nil, nil, nil, 0, err
 				}
-				*usedGas += params.CallValueTransferGas // In the future we may want to determine what a fair gas cost is
-				delete(etxSet, etxEntry.ETX.Hash())     // This ETX has been spent so remove it from the unspent set
+				*usedGas += params.CallValueTransferGas    // In the future we may want to determine what a fair gas cost is
+				totalEtxGas += params.CallValueTransferGas // In the future we may want to determine what a fair gas cost is
```

Comment:
can you create an issue to revisit this?

Reply:
https://github.com/orgs/dominant-strategies/projects/47/views/3?sliceBy%5Bvalue%5D=jdowning100&pane=issue&itemId=53137427

Added to this issue